.drone.star

```diff
@@ -25,6 +26,7 @@ def main(ctx):
     'squishversion': {
         'latest': '7.1.1-qt64x-linux64',
         'fedora': '7.2.1-qt66x-linux64',
+        'qt67'  : '8.0.0-qt67x-linux64',
```

Comment:
you can just update the version in fedora (client CI uses fedora build). This will create new `fedora-8.0.0-qt66x-linux64` and other docker tags.
And in the client ci, we are using specific tag. you might want to chage it in https://github.com/owncloud/client/pull/11994
https://github.com/owncloud/client/blob/83d7be7274e61c8c65aad50585f7bf3cbce35c9d/.drone.star#L26
```suggestion
        'fedora': '8.0.0-qt67x-linux64',
```

Reply:
This is what I explicitly didn't want to do. I want to keep the current fedora image untouched.

Reply:
This will create new tags `fedora-39-8.0.0-qt67x-linux64`, `fedora-8.0.0-qt67x-linux64` and will replace `fedora` tag
but we will still have old tags: `fedora-39-7.2.1-qt66x-linux64` and `fedora-7.2.1-qt66x-linux64`

Reply:
but then we can never recreate previous images .... feeö wrong to me ;-)

Reply:
yeah, we cannot.
we should also make sure that this craft target installs the matching qt version with squish qt (major.minor) https://github.com/owncloud-ci/squish/blob/16cad0f054419caf80c5e5916e525bd0f8b104cc/fedora/Dockerfile.amd64#L11

Reply:
we usually move on with the new updated versions but yeah, we cannot do patches to the old tags if we want to with the current setup.